src/otx/algo/modules/conv_module.py

```diff
@@ -124,9 +124,9 @@ def __init__(
         self,
         in_channels: int | tuple[int, ...],
         out_channels: int,
-        kernel_size: int | tuple[int, int],
-        stride: int | tuple[int, int] = 1,
-        padding: int | tuple[int, int] = 0,
+        kernel_size: int | tuple[int, int] | tuple[int, int, int],
```

Comment:
```suggestion
        kernel_size: int | tuple[int, ...],
```
Is it possible?

Reply:
fixed at 54eddd647e84e0cb272f687f467a8e4d10ededfb